### DISCUSSION TITLE
How to configure display python venv on the theme of ys like zsh

### DISCUSSION THREAD

**Discussion by zweix123** [JanDeDobbeleer/oh-my-posh#3463] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What would you like to see changed/added?

(The theme here is ys)
Through some [configurations](https://aber.sh/articles/ys-zsh-theme-virtualenv/), zsh can become the following  
![image](https://user-images.githubusercontent.com/74446314/216916035-dc696dc9-c9f4-4e11-a27a-2fc6903983d3.png)  
but what to do in posh？
![image](https://user-images.githubusercontent.com/74446314/216916601-3adf577b-3b7a-4ff7-9e15-b7a80fc88fd1.png)
This is ugly.

**Comment by HernandoR** [comment]
+1, facing the same problem.

should work if adding 
```
{
      "background": "#906cff",
      "foreground": "#100e23",
      "powerline_symbol": "\ue0b0",
      "style": "powerline",
      "template": " \ue235 {{ if .Error }}{{ .Error }}{{ else }}{{ if .Venv }}{{ .Venv }} {{ end }}{{ .Full }}{{ end }} ",
      "type": "python"
    },
```

to `blocks/segments` in theme file , but nothing displaying

**Comment by zweix123** [comment]
> +1，面临同样的问题。
> 
> 如果添加应该工作
> 
> ```
> {
>       "background": "#906cff",
>       "foreground": "#100e23",
>       "powerline_symbol": "\ue0b0",
>       "style": "powerline",
>       "template": " \ue235 {{ if .Error }}{{ .Error }}{{ else }}{{ if .Venv }}{{ .Venv }} {{ end }}{{ .Full }}{{ end }} ",
>       "type": "python"
>     },
> ```
> 
> 在`blocks/segments`主题文件中，但没有显示

What you don't understand is that the focus is not on the style, but on its display position

**Comment by zweix123** [comment]
I analyzed the source code of 'ys. omp. json' to solve the problem. In fact, it is very simple. The segments list in the json file is the sequence. We can move the configuration code belonging to python back to the appropriate location.
The final effect is as follows
![image](https://user-images.githubusercontent.com/74446314/216925209-f3dee4a5-84e7-4ad9-addc-760a8acceece.png)
like zsh!

**Comment by lewis-yeung** [comment]
@zweix123 On my end the value of the environment variable `VIRTUAL_ENV_DISABLE_PROMPT` is `1`, and the virtual environment name does not appear in front of the prompt.

**Comment by zweix123** [comment]
> @zweix123我这边环境变量的`VIRTUAL_ENV_DISABLE_PROMPT`值为`1`，提示符前没有出现虚拟环境名。

Yes, you are right.

**Comment by zweix123** [comment]
This is [my config](https://github.com/zweix123/posh-config)
